cmd/run.go

```diff
@@ -787,6 +786,9 @@ func bindRunFlags(cmd *cobra.Command) {
 	cmd.Flags().String("log-format", defaultConfig.Log.Format, "the log format to output logs in")
 	util.MustBindPFlag("log.format", cmd.Flags().Lookup("log-format"))
 
+	cmd.Flags().String("log-level", defaultConfig.Log.Level, "the log level to use")
```

Comment:
Mind documenting this in the `.config-schema.json` file as well?

Reply:
Always forget that. I wish we simply stuck with env vars 😢